.github/ISSUE_TEMPLATE/report-a-bug.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Report a bug
+about: Have you found a bug or issue? Create a bug report for this SDK
+title: ''
+labels: bug report
+assignees: ''
+---
+
+**Please do not report security vulnerabilities here**. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.
+
+**Thank you in advance for helping us to improve this library!** Please read through the template below and answer all relevant questions. Your additional work here is greatly appreciated and will help us respond as quickly as possible. For general support or usage questions, use the [Auth0 Community](https://community.auth0.com/) or [Auth0 Support](https://support.auth0.com/). Finally, to avoid duplicates, please search existing Issues before submitting one here.
+
+By submitting an Issue to this repository, you agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).
+
+### Description
+
+> Provide a clear and concise description of the issue, including what you expected to happen.
+
+### Reproduction
+
```

Comment:
Does it make sense to be clear that if the information is lacking in providing reproducible steps as we've asked for then we may not be able to follow up?

Reply:
I've added a note here, along with other bolder changes to the template.